src/index.css

```diff
@@ -1,4 +1,294 @@
+  opacity: 0.5;
+  color: #121212;
+}
+
+.booksList > li {
+  list-style: none;
+  border: 1px solid #e8e8e8;
+  margin-bottom: 0.938rem;
+  border-radius: 4px;
+  background-color: #fff;
+  padding: 2rem 9.1rem 1.75rem 1.68rem;
+  display: flex;
+  justify-content: space-between;
+}
+
+.bookContainer > span {
+  opacity: 0.5;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+  margin-bottom: 0.25rem;
+}
+
+.bookContainer > h2 {
+  font-family: 'Roboto Slab', serif;
+  font-size: 1.375rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.2px;
+  color: #121212;
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+.bookContainer > p {
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+  margin-bottom: 1.313rem;
+  margin-top: 0;
+}
+
+.actionsContainer > button {
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+  background-color: #fff;
+  padding-top: 0.188rem;
+  padding-bottom: 0.125rem;
+}
+
+.commentsBtn {
+  padding-right: 0.938rem;
+  border: none;
+  border-right: 1px solid #e8e8e8;
+}
+
+.removeBtn {
+  padding-left: 1.063rem;
+  padding-right: 0.938rem;
+  border: none;
+  border-right: 1px solid #e8e8e8;
+}
+
+.editBtn {
+  padding-left: 1.063rem;
+  border: none;
+}
+
+.progressContainer {
+  display: flex;
+  justify-content: space-between;
+  width: 23.625rem;
+  align-items: center;
+}
+
+.circleProgress {
+  display: flex;
+  align-items: center;
+  padding-right: 30px;
+  border-right: 1px solid #e8e8e8;
+}
+
+.circle {
+  height: 4.25rem;
+}
+
+.circleProgressText {
+  display: flex;
+  flex-direction: column;
+}
+
+.circleProgressText > p {
+  margin-top: 0;
+  margin-bottom: 0;
+  margin-left: 0.68rem;
+}
+
+.progressT1 {
+  font-family: 'Montserrat', sans-serif;
+  font-size: 2rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.progressT2 {
+  opacity: 0.5;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.875rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.chapterProgress {
+  display: flex;
+  flex-direction: column;
+}
+
+.chapterText {
+  opacity: 0.5;
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.813rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+  margin-top: 0;
+  margin-bottom: 0;
+}
```

Comment:
- [ ] Here as well,  kindly make sure to transform it to uppercase as given in the project guidelines for getting a replica of the Zeplin's design  📌🔨